hoodie-client/src/main/java/com/uber/hoodie/table/HoodieTable.java

```diff
@@ -201,22 +202,20 @@ public HoodieTimeline getCommitTimeline() {
   /**
    * Get only the completed (no-inflights) compaction commit timeline
    */
-  public HoodieTimeline getCompletedCompactionCommitTimeline() {
-    return getCompactionCommitTimeline().filterCompletedInstants();
+  public HoodieTimeline getCompletedCompactedCommitTimeline() {
```

Comment:
Similarly, just like for getCompactedCommitTimeline

Reply:
lets do `getCompletedCommitsTimeline`. meta point : if you need to refer to commit + delta commit, lets just call it `commits`